src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -2851,10 +2851,10 @@ private static bool IsAvailableFarEastCodePage(uint codePage)
         private static bool IsAnyDBCSCharSet(uint charSet)
         {
             const uint SHIFTJIS_CHARSET = 128;
-            const uint HANGEUL_CHARSET = 129;
+            const uint HANGUL_CHARSET = 129;
```

Comment:
Presumably `Hangule` (also fixed in this series) isn't accepted. I'll split this change out as well since it's adjacent to the `case`-`space` change.